### PR TITLE
[ENH] Output and load T2* maps in seconds

### LINKED ISSUE #252: T2* maps should be outputted in seconds, not milliseconds
### Summary
Per [BEP001](https://github.com/bids-standard/bep001/blob/e661031158c462c7c7ba9e11d42b99a67fca591e/src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md), T2* maps should be in units of seconds, not milliseconds.

### Additional Detail
BEP001 hasn't been merged yet, but it seems likely that the units for these kinds of maps will be set in stone. 

### Next Steps
1. Divide T2* estimates by 1000  when they're written out in the tedana and t2smap workflows.

### PR DESCRIPTION
**Pull request by tsalo**
Closes #252.

Changes proposed in this pull request:

- In `tedana_workflow` and `t2smap_workflow`, output limited and full T2* maps in seconds instead of milliseconds.
- In `tedana_workflow`, assume T2* maps used as inputs to the workflow are in seconds.
- In documentation, explicitly note that T2* outputs are in seconds.

### CODE REVIEW

File: tedana/workflows/t2smap.py [hunk 1]
```diff
@@ -216,7 +216,7 @@ def t2smap_workflow(data, tes, mask=None, fitmode='all', combmode='t2s',
     # anything that is 10x higher than the 99.5 %ile will be reset to 99.5 %ile
     cap_t2s = stats.scoreatpercentile(t2s_limited.flatten(), 99.5,
                                       interpolation_method='lower')
-    LGR.debug('Setting cap on T2* map at {:.5f}'.format(cap_t2s * 10))
+    LGR.debug('Setting cap on T2* map at {:.5f}ms'.format(cap_t2s * 10))
```
Comment (emdupre) [inline tedana/workflows/t2smap.py]: Should the logger report milliseconds if the output (and potential input) are in seconds ? Not sure if that will be confusing to an end-user.

Comment (tsalo) [reply tedana/workflows/t2smap.py]: Good thinking. Changed now.

**Comment by tsalo** [comment]
Coverage is technically increasing, and yet the CI is mad... Not sure why.

**Comment by tsalo** [comment]
Okay, now tests are finally passing.

**Comment by jbteves** [comment]
This is kind of nitpicky, but could we actually have a function like millis2seconds and seconds2millis or something to do this? It would reduce the amount of "magic numbers" in the code and avoid the need for your clarifying comments, while also being pretty straightforward to write.

**Comment by jbteves** [comment]
This is kind of nitpicky, but could we actually have a function like millis2seconds and seconds2millis or something to do this? It would reduce the amount of "magic numbers" in the code and avoid the need for your clarifying comments, while also being pretty straightforward to write.

**Comment by tsalo** [comment]
@jbteves Done. And I wrote tests for them.

I propose we ignore the CodeCov failure since the PR increases coverage.

File: tedana/workflows/t2smap.py [hunk 2]
```diff
@@ -231,9 +232,9 @@ def t2smap_workflow(data, tes, mask=None, fitmode='all', combmode='t2s',
     s0_limited[s0_limited < 0] = 0
     t2s_limited[t2s_limited < 0] = 0
 
-    io.filewrite(t2s_limited, op.join(out_dir, 't2sv.nii'), ref_img)
+    io.filewrite(t2s_limited / 1000, op.join(out_dir, 't2sv.nii'), ref_img)
```
Comment (jbteves) [inline tedana/workflows/t2smap.py]:
```suggestion
    io.filewrite(utils.millisec2sec(t2s_limited), op.join(out_dir, 't2sv.nii'), ref_img)
```

File: tedana/workflows/t2smap.py [hunk 3]
```diff
@@ -231,9 +232,9 @@ def t2smap_workflow(data, tes, mask=None, fitmode='all', combmode='t2s',
...
     io.filewrite(s0_limited, op.join(out_dir, 's0v.nii'), ref_img)
-    io.filewrite(t2s_full, op.join(out_dir, 't2svG.nii'), ref_img)
+    io.filewrite(t2s_full / 1000, op.join(out_dir, 't2svG.nii'), ref_img)
```
Comment (jbteves) [inline tedana/workflows/t2smap.py]:
```suggestion
    io.filewrite(utils.millisec2sec(t2s_full), op.join(out_dir, 't2svG.nii'), ref_img)
```

File: tedana/workflows/t2smap.py [hunk 4]
```diff
@@ -216,7 +216,8 @@ def t2smap_workflow(data, tes, mask=None, fitmode='all', combmode='t2s',
     # anything that is 10x higher than the 99.5 %ile will be reset to 99.5 %ile
     cap_t2s = stats.scoreatpercentile(t2s_limited.flatten(), 99.5,
                                       interpolation_method='lower')
-    LGR.debug('Setting cap on T2* map at {:.5f}'.format(cap_t2s * 10))
+    cap_t2s_sec = (cap_t2s * 10.) / 1000.
```
Comment (jbteves) [inline tedana/workflows/t2smap.py]:
```suggestion
    cap_t2s_sec = utils.millisec2sec(cap_t2s * 10.)
```

**Comment by jbteves** [comment]
I agree about ignoring codecov.

**Comment by tsalo** [comment]
🤦‍♂ Thanks for catching that. Great example of why I opened #488.

**Comment by jbteves** [comment]
It is! Okay, I like the changes now but why are e5519ac and dd5a5e6 on this PR? I'm confused.

**Comment by tsalo** [comment]
e5519ac is there because the new utils needed to be in the API page, and I noticed that there were other outdated links in that page while I was editing it. dd5a5e6 followed naturally from that, since `tedana.metrics.__init__` had functions that came from `tedana.stats`.

**Pull request merged**